lib/src/csv_parser.cpp

```diff
@@ -103,8 +105,30 @@ Problem CsvParser::load_input() const {
     return problem;
 }
 
-void CsvParser::save_output(const Solution& sln) const {
-    UNUSED(sln)
+void CsvParser::save_output(const Problem& prb, const Solution& sln) const {
+	std::ofstream outfile("solution.csv");
+
+	outfile << "Route;" << "Vehicle;" << "Customer;" << "Arrive;" << "Begin;" << "End;" << "Leave;" << "Distance from previous;" << "Depot distance;" << std::endl;
```

Comment:
If you meant ";" at line 111, for some reason the output was not divided into columns, so we had to use ";" instead.

Reply:
@juggeroid no, I mean that there's no need for a final `;` because there's a `\n` right after it:

    simple csv:
    value1;value2;value3\n (no need for `;` at the end)
    value4;value5;value6...

by the way, we decided to use `;` instead of `,` due to the fact that `MS Excel` expects `;` as a CSV delimiter by default instead of `,` (which doesn't make sense to me since CSV is literally "comma separated values")